src/pal/tests/palsuite/runpaltests.sh

```diff
@@ -38,16 +38,35 @@ PAL_TEST_LIST=$PWD/paltestlist.txt
 cd $OLDPWD
 echo The list of PAL tests to run will be read from $PAL_TEST_LIST
 
-# Create path to a folder where PAL tests will store their output
-# First check if the output folder was passed as a parameter to the script.
-# It is supposed be the second parameter so check if we have more than 1 argument.
+# Create the test output root directory
+mkdir -p /tmp/PalTestOutput
```

Comment:
This isn't the directory tests actually use as their working folder. It's just the root directory under which a unique temp directory is created below with mktemp. The PalTestOutput directory needs to exist for mktemp to succeed creating a directory under it. Otherwise, I was just preserving the previous behavior to delete PalTestOutput if it exists but is not a directory, which shouldn't happen more than once, and typically should never happen.